kuma/dashboards/views.py

```diff
@@ -130,3 +132,19 @@ def topic_lookup(request):
     data = json.dumps(topiclist)
     return HttpResponse(data,
                         content_type='application/json; charset=utf-8')
+
+
+@require_POST
+@permission_required('wiki.add_revisionakismetsubmission')
+def submit_akismet_spam(request):
+    """Creates SPAM or HAM Akismet record for revision"""
+    url = request.POST.get('next')
+    if url is None or not is_safe_url(url, request.get_host()):
+        url = reverse('dashboards.revisions')
```

Comment:
I'll add this test now. If the URL is empty or invalid, the user will still be able to submit spam successfully, it will just go back to the dashboard with no filters/page set.